cli/src/commands/edit.rs

```diff
@@ -25,6 +25,8 @@ use crate::ui::Ui;
 ///
 /// Note: it is generally recommended to instead use `jj new` and `jj
 /// squash`.
+///
+/// For more information, see https://github.com/martinvonz/jj/blob/main/docs/FAQ.md#how-do-i-resume-working-on-an-existing-change
```

Comment:
I think we prefer to link to https://martinvonz.github.io/jj/latest instead

Reply:
Will fix